spec.bs

```diff
@@ -743,6 +752,11 @@ authentication:
     otherwise the [[#sctn-steps-to-check-if-a-payment-can-be-made|the steps
     to check if a payment can be made]] would have failed previously.
 
+The user agent MAY utilize the information in
+{{SecurePaymentConfirmationRequest/localeHint}}, if any, to display a UX in a
+matching language and locale-based formatting to that which the website is
+using for its interactions with the user.
+
```

Comment:
Proposed:

"The user agent MAY utilize the information in {{SecurePaymentConfirmationRequest/localeHint}}, if any, to match the website's language and locale in the user agent's UX."

Reply:
I used Addison's language here, please take another look and let me know if you have additional changes in mind :)